Yoast/YoastSEO.js
# Change all usage of `lodash` to `lodash-es`.

## Summary

This PR can be summarized in the following changelog entry:

* Change all usage of `lodash` to `lodash-es`.

## Relevant technical choices:

* This allows YoastSEO.js to be tree-shaken better.

## Test instructions

This PR can be tested by following these steps:

* Make sure all the tests still run
* Make sure the analysis works as expected.

Fixes #1736
